### DISCUSSION TITLE
Recognize Tuya Silvercrest Devices | Tuya-SmartLife | LIDL

### DISCUSSION THREAD

**Discussion by weberhannis** [milo526/homebridge-tuya-web#213] [Q&A]
The discounter LIDL offers numerous Tuya products. These work with the Lidl app and also with the SmartLife app. Unfortunately, the Homebride plugin does not recognize these devices. 

https://www.lidl.de/de/silvercrest-steckdose-zwischenstecker-zigbee-smart-home/p355168

Other devices connected to the SILVERCREST bridge are easily detected.

Is there a way to solve the problem?

Regards

**Comment by milo526** [comment]
Sadly there is not.

Lidl uses zigbee devices.
This plugin exclusively supports WiFi devices.

**Reply by weberhannis** [reply]
Hi Milo,
thanks for your quick response.

I have two devices osram led stripe and mueller light lamp at the zigbee-bridge that are recognized. There is no additional client IP in my Wi-Fi home network for these devices. The zigbee-bridge ist connected via LAN cable to fritzbox.  But now I will check again deeply.  Thank you for your comment.

I think - well I hope - Tuya brings together all the devices and protocols on its platform, Zigbee, Wi-Fi, Z-Wave, ... and publishes them via a uniform interface. But of course, you have already programmed at this point and you know the internal structures more precisely.

I will come back and report when my analysis is complete.
